test/units/plugins/action/test_gather_facts.py

```diff
@@ -22,8 +22,9 @@
 from units.compat.mock import MagicMock, patch
 
 from ansible import constants as C
-from ansible.plugins.action.gather_facts import ActionModule
 from ansible.playbook.task import Task
+from ansible.plugins.action.gather_facts import ActionModule
```

Comment:
```suggestion
from ansible.plugins.action.gather_facts import ActionModule as GatherFactsAction
```
makes it clearer in the code below what's going on